Source/ACE/Factories/SpellFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ACE.Entity;
+using ACE.Entity.Enum;
+using ACE.Network.Enum;
+
+namespace ACE.Factories
+{
+    /// <summary>
+    /// factory class for creating objects related to administration or content-recreation
+    /// </summary>
+    public class SpellFactory
+    {
+        /// <summary>
+        /// creates a spell with velocity directly in front of the player's position provided
+        /// </summary>
+        public static ImmutableWorldObject CreateCastedSpell(ushort weenieClassId, Position newPosition, ObjectGuid objectid, AceVector3 velocity)
+        {
+
+            // all we need right now ..  + required..
+            var weenie = WeenieHeaderFlag.Spell;
+
+
+            ImmutableWorldObject wo = new ImmutableWorldObject(ObjectType.None, objectid, "Ball of Death", weenieClassId, ObjectDescriptionFlag.None, weenie, newPosition);
+
+            // flame bolt
+            wo.GameData.Spell = (ushort)27;
```

Comment:
Make a spells enum instead of hard-coding 27.  This data existed in Decal before, so I'm sure we can get it now.